lib/recurly/paypal/braintree.js

```diff
@@ -89,24 +86,26 @@ export class BraintreePayPal extends Emitter {
       }
 
       // Tokenize with Recurly
-      this.recurly.request('post', '/paypal/token', { payload }, (error, token) => {
+      this.recurly.request('post', '/paypal/token', { type: 'braintree', payload }, (error, token) => {
         if (error) return this.error('paypal-braintree-tokenize-recurly-error', { error });
         this.emit('token', token);
       });
     });
   }
 
   /**
-   * Creates and emits a RecurlyError
+   * Falls back to non-Braintree PayPal integration
    *
-   * @param  {...Mixed} params to be passed to the Recurlyerror factory
-   * @return {RecurlyError}
-   * @emit 'error'
-   * @private
+   * @param  {String} reason
+   * @param  {Object} options
+   * @return {PayPal}
    */
-  error (...params) {
-    let err = params[0] instanceof Error ? params[0] : errors(...params);
-    this.emit('error', err);
-    return err;
+  fail (reason, options) {
+    debug('Failure scenario encountered. Falling back to direct PayPal flow', reason, options);
+    let failure = this.failure = {};
+    if (reason) failure.error = this.error(reason, options);
+    failure.fallback = new PayPal(omit(this.config.constructorOptions, 'braintree'));
+    this.emit('failure', failure);
+    return failure;
```

Comment:
Failure can occur synchronously and asynchronously. In both cases a `failure` event produces the fallback instance.